src/demo/index.jsx

```diff
@@ -1,43 +1,32 @@
 import $ from 'jquery';
-import constants, {Modes} from './constants';
+import {Modes} from './constants';
 import {init as initRenderer} from './renderer';
 import {init as initTraining} from './modes/training';
 import {init as initPredicting} from './modes/predicting';
 import {init as initPond} from './modes/pond';
 import {setState, getState} from './state';
+import {generateRandomFish} from '../utils/generateOcean';
 
 $(document).ready(() => {
-  let canvas = document.getElementById('activity-canvas');
-  canvas.width = constants.canvasWidth;
-  canvas.height = constants.canvasHeight;
+  // Generate some fish
+  let fishes = [];
+  for (let i = 0; i < 100; i++) {
```

Comment:
nit: you could import `generateOcean` and use that here. Does exactly this right now but I figure we can add any logic to generate 15% red fish or whatever in that function.

Reply:
Oh ah you can't do that. Then I will try and add a better helper method. :)